### PR TITLE
feat(language-service): provide completion for $event variable

### PR DESCRIPTION
**Pull request by ayazhafiz**
This commit adds a completion for the `$event` variable in bound event
expressions.

This is the first in a series of PRs to support completions for
properties on `$event` variables (https://github.com/angular/vscode-ng-language-service/issues/531).

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Feature

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/language-service/src/expression_diagnostics.ts [hunk 1]
```diff
@@ -25,8 +25,7 @@ export interface DiagnosticTemplateInfo {
 
 export function getTemplateExpressionDiagnostics(info: DiagnosticTemplateInfo): Diagnostic[] {
   const visitor = new ExpressionDiagnosticsVisitor(
-      info, (path: TemplateAstPath, includeEvent: boolean) =>
-                getExpressionScope(info, path, includeEvent));
+      info, (path: TemplateAstPath, includeEvent: boolean) => getExpressionScope(info, path));
```
Comment (kyliau) [inline packages/language-service/src/expression_diagnostics.ts]: remove `includeEvent`

**Review by kyliau — CHANGES_REQUESTED** [review]

**Comment by kyliau** [comment]
Could you please rerun circle CI? the test status seems to be stale

**Comment by kyliau** [comment]
```
packages/language-service/src/completions.ts:337:83 - error TS2554: Expected 2 arguments, but got 3.

337       new ExpressionVisitor(info, position, () => getExpressionScope(dinfo, path, false));
```

**Comment by ayazhafiz** [comment]
Forgot about that. Thanks.

**Comment by ayazhafiz** [comment]
@kyliau ptal

**Review by kyliau — APPROVED** [review]

**Comment by atscott** [comment]
Hi @ayazhafiz, can you rebase and resolve the merge conflict? Thanks!

**Comment by atscott** [comment]
Sorry @ayazhafiz! I merged another PR that conflicted with this one. Can you rebase again?

**Pull request closed without merging**